BSP/STM32F413/Src/main.c

```diff
@@ -0,0 +1,16 @@
+ /* Program for UTSVT BeVolt's Battery Protection System*/
+ 
+#include "BSP_CAN.h"
+#include "config.h"
+
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#define BSPVARIABLE 0x10b
```

Comment:
Probably calL this something more descriptive like VOLTAGECANID

Reply:
Yes, definitely. This is a bad name for a `#define`, since it tells you nothing about what it is used for (and it is not even a variable).